helium_miner_dashboard.json

```diff
@@ -562,14 +562,14 @@
       "targets": [
         {
           "exemplar": true,
-          "expr": "rate(node_disk_io_time_weighted_seconds_total{instance=\"$server_name\",device!~\"^sd[ab]\"}[5m])",
+          "expr": "rate(node_disk_io_time_weighted_seconds_total{instance=~\"$server_name:.*\",device!~\"^sd[ab]\"}[5m])",
```

Comment:
do you think the 'device' part is to idiosyncratic? My miner has ~5 disks, so this was to reduce the noise on my graph. Argument for leaving it is to show that it's possible, argument for omitting it is because by definition, it's wrong for 99% of installs.

Reply:
I think you should remove it. But I'm part of the 99% so I'm being selfish!

Showing it is possible is nice, but I think this repo is valuable as a  "here are some ideas on the set of metrics you might want to look at, should work for 99%".  It doesn't have to show every possible query feature. As an example, I didn't do this PR just to show people how to do it, but because I think nearly all of us are going to want the node_exporter panels right next to miner_exporter ones  and making sure both variables were in sync bugged me. 

Personally, I run my own customized version of this with a different filter for devices (my AWS instances have md0, xvda, and xvdb. md0 is useless for me to look at) and I also get rid of the fan speed panel